sunpy/time/utime.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from astropy.time.formats import erfa, TimeFromEpoch
+
+
+class TimeUTime(TimeFromEpoch):
+    """Seconds from 1979-01-01 00:00:00 UTC. Same as Unix time
+    but this starts 9 years later.
+
+    This time format is included for historical reasons. Some
+    people in solar physics prefer using this epoch.
+
+    Example:
+        from astropy.time import Time
+        t = Time('2000-01-01T13:53:23')
+        print(t.utime)
+        >>> 662738003.0
```

Comment:
This looks wrong for example syntax. 

I think it should be somrthing like:
```
Examples
--------
     >>> code line 1
     >>> code line 2
```
I forget how you add the output.

Reply:
I think you're right. I couldn't find a PEP standard but I found a [random example of how it's done in scipy](https://github.com/scipy/scipy/blob/master/scipy/interpolate/_bsplines.py). For example: 
```
 >>> from scipy.interpolate import BSpline
 >>> k = 2
 >>> t = [0, 1, 2, 3, 4, 5, 6]
 >>> c = [-1, 2, 0, -1]
 >>> spl = BSpline(t, c, k)
 >>> spl(2.5)
 array(1.375)
 >>> bspline(2.5, t, c, k)
 1.375
```